includes/Shortcode/Stories_Shortcode.php

```diff
@@ -0,0 +1,131 @@
+
+/**
+ * Class Stories_Shortcode
+ *
+ * @package Google\Web_Stories\Shortcode
+ */
+class Stories_Shortcode {
+
+	/**
+	 * Shortcode attributes.
+	 *
+	 * @var array
+	 */
+	private $attributes;
+
+	/**
+	 * Shortcode name.
+	 *
+	 * @var string
+	 */
+	const SHORTCODE_NAME = 'stories';
+
+	/**
+	 * Initializes the Stories shortcode.
+	 *
+	 * @return void
+	 */
+	public function init() {
+		add_shortcode( self::SHORTCODE_NAME, [ $this, 'render_stories' ] );
+	}
+
+	/**
+	 * Callback for the shortcode.
+	 *
+	 * This will render the stories according to given
+	 * shortcode attributes.
+	 *
+	 * @param array $attributes Shortcode attributes.
+	 *
+	 * @return string Story markup.
+	 */
+	public function render_stories( array $attributes ) {
+		$this->attributes = shortcode_atts(
+			[
+				'view'                      => 'circles',
+				'columns'                   => 1,
+				'title'                     => 'false',
+				'author'                    => 'false',
+				'date'                      => 'false',
+				'story_poster'              => 'true',
+				'archive_link'              => 'false',
+				'archive_label'             => __( 'View all stories', 'web-stories' ),
+				'list_view_image_alignment' => 'left',
+				'class'                     => '',
+				'number'                    => 10,
+				'order'                     => 'DESC',
+			],
+			$attributes
+		);
+
+		$stories = new Stories( $this->prepare_story_attrs(), $this->prepare_story_args() );
+
+		return $stories->render();
+	}
+
+	/**
+	 * Prepare story attributes.
+	 *
+	 * @return array Attributes to pass to Story_Query class.
+	 */
+	private function prepare_story_attrs() {
+		$attrs = [];
+
+		$attrs['view_type']                 = (string) $this->attributes['view'];
+		$attrs['number_of_columns']         = (int) $this->attributes['columns'];
+		$attrs['show_title']                = (bool) 'true' === $this->attributes['title'];
+		$attrs['show_author']               = (bool) 'true' === $this->attributes['author'];
+		$attrs['show_date']                 = (bool) 'true' === $this->attributes['date'];
+		$attrs['show_story_poster']         = (bool) 'true' === $this->attributes['story_poster'];
+		$attrs['show_story_archive_link']   = (bool) 'true' === $this->attributes['archive_link'];
+		$attrs['show_story_archive_label']  = (bool) 'true' === $this->attributes['archive_label'];
+		$attrs['list_view_image_alignment'] = (string) $this->attributes['list_view_image_alignment'];
+		$attrs['class']                     = (string) $this->attributes['class'];
+
+		return $attrs;
+	}
+
+	/**
+	 * Prepare story arguments.
+	 *
+	 * @return array Array of story arguments to pass to Story_Query.
+	 */
+	private function prepare_story_args() {
+		$args = [];
+
+		// Show 100 stories at most to avoid 500 errors.
+		$args['posts_per_page'] = min( (int) $this->attributes['number'], 100 );
+		$args['order']          = 'ASC' === $this->attributes['order'] ? 'ASC' : 'DESC';
+
+		return $args;
```

Comment:
```suggestion
		return [
		    	// Show 100 stories at most to avoid 500 errors.
		       'posts_per_page' => min( (int) $this->attributes['number'], 100 ),
		       'order'  => 'ASC' === $this->attributes['order'] ? 'ASC' : 'DESC', 
		];
```

The args local variable seems a little pointless here. Why not something simpler like this